featurebyte/schema/feature_list_namespace.py

```diff
@@ -18,6 +19,7 @@ class FeatureListNamespaceModelResponse(FeatureListNamespaceModel):
     """
 
     readiness_distribution: FeatureReadinessDistribution
+    dtype_distribution: List[FeatureTypeFeatureCount]
     primary_entity_ids: List[PydanticObjectId]
     entity_ids: List[PydanticObjectId]
     table_ids: List[PydanticObjectId]
```

Comment:
This PR (https://github.com/featurebyte/featurebyte/pull/1896/files) removes 
```
    dtype_distribution: List[FeatureTypeFeatureCount] = Field(allow_mutation=False)
    entity_ids: List[PydanticObjectId] = Field(allow_mutation=False)
    table_ids: List[PydanticObjectId] = Field(allow_mutation=False)
```
from feature list namespace model. We should add these attributes here to keep feature list namespace model response the same.